src/store/reducers/TodosSlice.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { ITodo } from '../../models/ITodo';
+
+type TodosState = {
+  todos: ITodo[];
+  isLoading: boolean;
+  error: string;
+};
+
+const initialState: TodosState = {
+  todos: [],
+  isLoading: false,
+  error: '',
+};
+
+export const getAllTodos = createAsyncThunk(
+  'todos/getAll',
+  async (_, { rejectWithValue }) => {
+    try {
+      const response = await axios.get<ITodo[]>('https://mate.academy/students-api/todos');
```

Comment:
why not using fetch here?

Reply:
@aliceblunt8 using axios just for testing purpose